### ISSUE TITLE
Not able to render anything in anyway in Android

### ISSUE THREAD

**Issue by witalobenicio** [satya164/react-native-tab-view#1352]
### Current behavior

I'm always getting this error:

`Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.`

This started to happen after update `react-native-reanimated` that lead me to update this lib because of another error (related to `el.getNode`).

```javascript
<TabView
          {...this.props}
          navigationState={{
            index: 0,
            routes: [
              { key: 'follow', title: Strings.IN_PROGRESS },
              { key: 'done', title: Strings.FINISHEDS },
            ],
          }}
          renderScene={SceneMap({
            follow: () => (
              <View>
                <Text>Witalo</Text>
              </View>
            ),
            done: () => (
              <View>
                <Text>Witalo</Text>
              </View>
            ),
          })}
          onIndexChange={this.changeIndex}
          initialLayout={{ width: Dimensions.get('window').width }}
          renderTabBar={this.renderTabBar}
        />
```

I've also tried by not using SceneMap, in fact this is the way that was working before.

### Expected behavior

 This should show something in the tabs

### Reproduction

I have no link

### Platform

- [X] Android
- [ ] iOS
- [ ] Web
- [ ] Windows
- [ ] MacOS

### Environment

| package                                | version |
| -------------------------------------- | ------- |
| react-native-tab-view                  | 3.0.0 / 3.1.1
| react-native-pager-view                | 6.0.0-rc.0
| react-native                           | 0.66.1
| expo                                   | not using
| node                                   | v14
| npm or yarn                            | 1.22.17

**Comment by witalobenicio** [comment]
Anyone???

**Comment by witalobenicio** [comment]
Fixed!

The problem was the version of `react-native-pager-view`. Anyone having this problem, just update to latest.

**Comment by WNemencha** [comment]
Had the same issue by following the readme where it mention I should install `react-native-pager-view@next`, which lead yarn to install it at `6.0.0-rc.2`. Fixed by downgrading to 5.4.23 as said by the cool bot!
